### PR TITLE
Replace disrespectful terms - 3

### PR DESCRIPTION
**Pull request by grac3gao**
Fixes #
related #1862

## Proposed Changes

-  It is suggested to not "health check", and provided preferred terms: check, confidence check, initial check, readiness check. However, no preferred terms are close to the meaning of the "health check" here. As the "health check" is connecting to the readiness probe and liveness probe in the Pod, change the name to be probe check.
- 
_

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by grac3gao** [comment]
Code Style check failed for reason:
```
Please consider avoiding tail format like this:
- t.Logf(Failed to execute readiness and liveness check: %v, err) (suggested to be changed like this)
+ t.Logf("Failed to execute readiness and liveness check: %v", err) (what I have)
```
The suggestion is not an accepted format for `t.Logf`. There might be something wrong in the Code Style check.

**Comment by tommyreddad** [comment]
@grac3gao I believe it should be recommending to use `t.Log("Failed to execute readiness and liveness check:", err)`. The linting output doesn't seem right.

**Comment by grac3gao** [comment]
> @grac3gao I believe it should be recommending to use `t.Log("Failed to execute readiness and liveness check:", err)`. The linting output doesn't seem right.

@tommyreddad I agree so.

File: cmd/broker/fanout/main.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func main() {
 	if err != nil {
 		logger.Fatal("Failed to create fanout sync pool", zap.Error(err))
 	}
-	if _, err := handler.StartSyncPool(ctx, syncPool, syncSignal, env.MaxStaleDuration, handler.DefaultHealthCheckPort); err != nil {
+	if _, err := handler.StartSyncPool(ctx, syncPool, syncSignal, env.MaxStaleDuration, handler.DefaultReadinessAndLivenessCheckPort); err != nil {
```
Comment (yolocs) [inline cmd/broker/fanout/main.go:102]: I feel it's a bit verbose to say "ReadinessAndLiveness". How about just ProbeCheck?

File: test/test_images/probe_helper/probe_helper_test.go [hunk 2]
```diff
@@ -548,37 +548,37 @@ func makeProbeHelper(ctx context.Context, t *testing.T, group *errgroup.Group) m
...
+func assertCheckResult(t *testing.T, url string, ok bool) {
 	req, err := http.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
-		t.Fatalf("Failed to create health check request: %v", err)
+		t.Fatalf("Failed to create liveness check request: ", err)
 	}
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		t.Logf("Failed to execute health check: %v", err)
+		t.Logf("Failed to executeliveness check: ", err)
```
Comment (tommyreddad) [inline test/test_images/probe_helper/probe_helper_test.go:573]: Missing a space.
```suggestion
		t.Logf("Failed to execute liveness check: ", err)
```

File: cmd/broker/fanout/main.go (hunk 1 above)
Comment (grac3gao) [reply cmd/broker/fanout/main.go:102]: That's a fantastic suggestion! I like the name. will change to it

**Comment by grac3gao** [comment]
```
  Please consider avoiding tail format like this:
  - t.Logf(Failed to execute probe check: %v, err)
  + t.Log("Failed to execute probe check:", err)
```
which can't pass unit and build test. https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/google_knative-gcp/1916/pull-google-knative-gcp-build-tests/1327339059076403200, https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/google_knative-gcp/1916/pull-google-knative-gcp-unit-tests/1327339059101569024

**Pull request merged**